src/test/unit/DelegationUnit.t.sol

```diff
@@ -6619,6 +6539,8 @@ contract DelegationManagerUnitTests_completeQueuedWithdrawal is DelegationManage
                 strategyMock, 
                 depositSharesPerWithdrawal
             );
+            withdrawal.nonce = nonce;
+            nonce += 1;
```

Comment:
Note: this was needed because it was looping over creating queuedWithdrawalParams but each one was having a nonce of 0 from reading directly off the contract. I overwrite this value here since the expectation is that the withdrawals are queued in order with nonces from 0-3 inclusive